### PR TITLE
Add final name "Crime-Database"

### LINKED ISSUE #51: Add final name for project

### PR DESCRIPTION
**Pull request by FelixJacobsen**
Fix #51 

Adding final name in pom.xml

### CODE REVIEW

**Comment by helenahalldiniths** [comment]
When adding the final name, maybe we should update the dockerfile as well? (So it works :) )

**Comment by LordRekishi** [comment]
Do we need a final name? And is there another way to implement it in Spring Boot? I get the feeling this is not the right way? OR did we talk about this during class?

**Comment by FelixJacobsen** [comment]
Finalname has nothing to do with Spring boot I think, it's more to have a static name for deployment/tests? 

image

**Comment by LordRekishi** [comment]
> Finalname has nothing to do with Spring boot I think, it's more to have a static name for deployment/tests?
> 
> image

Should it be scrapped then? Do we get any benefit from it?

**Comment by helenahalldiniths** [comment]
The docker file uses "CrimeDatabase-0.0.1-SNAPSHOT.jar ", which we will have to change during the lifetime of the application (When we make a release we will remove SNAPSHOT and then increase the number). If we have a final name we don't have to remember to update the dockerfil each time we update the version of the project :)

File: pom.xml [hunk 1]
```diff
@@ -39,6 +39,7 @@
     </dependencies>
 
     <build>
+        <finalName>Crime-Database</finalName>
```
Comment (LordRekishi) [inline pom.xml]:
```suggestion
        <finalName>crime-database</finalName>
```

**Review by LordRekishi — CHANGES_REQUESTED** [review]

**Review by LordRekishi — APPROVED** [review]

**Pull request merged**